packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -68,19 +78,24 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     /// @param _vm An onchain VM that performs single instruction steps on a fault proof program
     ///            trace.
     /// @param _l2oo The trusted L2OutputOracle contract.
-    /// @custom:semver 0.0.4
+    /// @param _blockOracle The block oracle, used for loading block hashes further back
+    ///                     than the `BLOCKHASH` opcode allows as well as their estimated
+    ///                     timestamps.
+    /// @custom:semver 0.0.6
```

Comment:
Why did we bump this two patch versions?

Reply:
This PR was stacked, the other one was merged into this due to overlap.